sw/host/rom_ext_image_signer/Cargo.lock

```diff
@@ -0,0 +1,210 @@
+# This file is automatically @generated by Cargo.
```

Comment:
I don't believe Cargo.lock files are usually checked in. Are we doing this on purpose, or should we fix the gitignore?

Reply:
From [Cargo.toml vs Cargo.lock (docs.rust-lang.org)](https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html)
> If you’re building an end product, which are executable like command-line tool or an application, or a system library with crate-type of staticlib or cdylib, check Cargo.lock into git.